src/pipdeptree/_models/package.py

```diff
@@ -83,22 +84,13 @@ def as_frozen_repr(obj: DistInfoDistribution) -> str:
         # pip._vendor.pkg_resources.DistInfoDistribution.
         #
         # This is a hacky backward compatible (with older versions of pip) fix.
-        try:
-            from pip._internal.operations.freeze import FrozenRequirement  # noqa: PLC0415 # pragma: no cover
-        except ImportError:
-            from pip import FrozenRequirement  # type: ignore[attr-defined, no-redef] # noqa: PLC0415 # pragma: no cover
+        if not obj.key:
+            return ""
```

Comment:
Why is this here?

Reply:
I think it is because what I mentioned here https://github.com/tox-dev/pipdeptree/pull/333#discussion_r1528623653 .
As I replied, I will remove this either.